property_test.go

```diff
@@ -25,7 +25,7 @@ func TestGetPropertyNameSelectorExpr(t *testing.T) {
 		"string",
 		"",
 	}
-	propertyName, err := getPropertyName(input, New())
+	propertyName, err := getPropertyName("test", input, New())
```

Comment:
I guess you're testing it so hardcode "test" string, right? 😄

Reply:
What do you suggest?